### ISSUE TITLE
Breaks when resizing console during download

### ISSUE THREAD

**Issue by deevus** [lukesampson/scoop#533]
```
updating openjdk (1.7.0-u60 -> 1.7.0-u80)
uninstalling openjdk (1.7.0-u60)
installing openjdk (1.7.0-u80)
downloading https://bitbucket.org/alexkasko/openjdk-unofficial-builds/downloads/openjdk-1.7.0-u80-unofficial-windows-amd64-image.zip...57%Exception calling "SetCursorPosition" with "2" argument(s): "The value must be greater than or equal to zero and less than
the console's buffer size in that dimension.
Parameter name: left
Actual value was 135."
At C:\Users\Simon\appdata\local\scoop\apps\scoop\current\lib\install.ps1:148 char:17
+                 [console]::setcursorposition($left, $top)
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentOutOfRangeException

Exception calling "SetCursorPosition" with "2" argument(s): "The value must be greater than or equal to zero and less than
the console's buffer size in that dimension.
Parameter name: left
Actual value was 135."
At C:\Users\Simon\appdata\local\scoop\apps\scoop\current\lib\install.ps1:162 char:5
+     [console]::setcursorposition($left, $top)
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentOutOfRangeException
```

**Comment by rrelmy** [comment]
This is fixed, maybe by #1329

**Comment by vsviridov** [comment]
Just caught this today:
![image](https://github.com/ScoopInstaller/Scoop/assets/221684/6931d248-e1c2-4bfa-bd9b-74908a5fbf6c)
